src/main/java/com/savvato/tribeapp/repositories/ToBeReviewedRepository.java

```diff
@@ -10,7 +10,8 @@
 
 @Repository
 public interface ToBeReviewedRepository extends CrudRepository<ToBeReviewed, Long> {
-    @Query(nativeQuery = true, value = "select tbr.* from to_be_reviewed tbr where tbr.id>?1 and tbr.has_been_groomed=1")
+
+    @Query(nativeQuery = true, value = "select tbr.* from to_be_reviewed tbr where tbr.id = (select min(tbr.id) from to_be_reviewed tbr where tbr.id>?1 and tbr.has_been_groomed=1)")
```

Comment:
I think this query, and the way it was before, are effectively the same... I think? Tell me, what inspired this change?